orchestration/orchestration.go

```diff
@@ -180,15 +180,41 @@ func (o *Orchestrator) DeleteService(ctx context.Context, input *ServiceDeleteIn
 func (o *Orchestrator) processCluster(ctx context.Context, input *ServiceOrchestrationInput) (*ecs.Cluster, error) {
 	client := o.ECS
 	if input.Service.Cluster != nil {
-		log.Infof("Using provided cluster name %s", aws.StringValue(input.Service.Cluster))
+		log.Infof("Using provided cluster name %s", aws.StringValue(input.Cluster.ClusterName))
```

Comment:
weird, why didnt this blow up before?

Reply:
Need to revisit this conditional against input.Service.Cluster - it seems it never fires.